SU-HSDO/suhumsci
# HSD8-711 blurry lazy image

# READY FOR REVIEW (no rush)

# Summary
- Use a 25x25 image as the initial image, then lazy load past that
- cleaned up some deprecated function use

# Need Review By (Date)
- 10/8

# Urgency
- low

# Steps to Test
1. checkout branch
1. `drush cim -y; drush cr`
1. create a page with a wysiwyg
1. in the wysiwyg upload a large-ish image and embed it using the "Large" image style
1. save the page. verify at first load it loads a very blurry image, then lazy loads the good image.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
